src/data-capsule.js

```diff
@@ -1,11 +1,14 @@
 import BaseStorage from './base-storage';
+import { verifyConsentPolicyCategoryIfExists } from './consent-policy';
 
 function validateNamespace(options) {
   if (!options.namespace) {
     throw new Error('namespace is required');
   } else if (typeof options.namespace !== 'string') {
     throw new Error('namespace should be a string');
   }
+
+  verifyConsentPolicyCategoryIfExists(options.category);
```

Comment:
If `category` is (about to be) an expected field in `options`, it should be added to `Options` in `src/typings/index.d.ts`.

Reply:
Correct, I guess I just forgot:) but let’s call it a feature - afaik it shouldn’t be exposed _yet_.